l10n_ch_account_statement_base_import/view/statement_view.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This view is not needed anymore. It references old field (already_completed) which does not exist in v8 and adds a button (download_attachment) that has no corresponding code to be called. Please remove this file and its reference in openerp manifest.